<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XLStatusBar</class>
 <widget class="QWidget" name="XLStatusBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="liveTimeLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="recordTimeLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="cpuLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dropFrameLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="indent">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_13">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>495</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="networkSpeedIconLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="networkSpeedLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
